src/App.tsx

```diff
@@ -459,10 +460,10 @@ function App(): ReactElement {
   );
 
   const updateFeature = useCallback(
-    async (newDataset: Dataset, newFeatureName: string): Promise<void> => {
+    async (newDataset: Dataset, newFeatureName: string): Promise<string> => {
```

Comment:
Does this need to be `async` at all? I don't see any `await` points

Reply:
Good catch, it doesn't.